modules/Bio/EnsEMBL/Production/Pipeline/Webdatafile/WebdataFileBootstrap.pm

```diff
@@ -0,0 +1,226 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+ Bio::EnsEMBL::Production::Pipeline::Webdatafile::WebdataFileBootstrap;
+
+=head1 DESCRIPTION
+  Compute bootstrap step for webdatafile dumps
+
+=cut
+
+package Bio::EnsEMBL::Production::Pipeline::Webdatafile::WebdataFileBootstrap;
+
+use strict;
+use warnings;
+
+use Bio::EnsEMBL::Registry;
+use base qw/Bio::EnsEMBL::Production::Pipeline::Common::Base/;
+use Path::Tiny qw(path);
+use Array::Utils qw(intersect);
+use Path::Tiny qw(path);
+use Carp qw/croak/;
```

Comment:
I don't think croak is used anywhere so could be deleted. In any case, in a hive module context, it's better to use $self->warning() - that gets logged in the hive db, whereas croaks would go to STDOUT, and effectively be lost if run via beekeeper.

Reply:
ok